### PR TITLE
fix: Ensures the submenu els are present before attempting to display them

### PR DESCRIPTION
**Pull request by gdixon**
##### Description

This PR solves the navbar.js issue --- the mobile submenu implementation has not been rolled out over the products menu, so the errors are coming from users hitting one of the product links and navigating away, the href works, but the errors get thrown before the browser navigates.

This will place the checks in the right place to make sure the sub-submenu elements are present before attempting to place the menu in the correct position.

##### Refers/Fixes

Refers: #9144

Fixes: [Cannot read property 'classList' of null - Events: 1.1k ~ Users: 594](https://sentry.io/organizations/gitcoin/issues/2464485024/?project=1398424&query=is%3Aunresolved+assigned_to%3Ame&statsPeriod=14d)
Fixes: [computedRootStyles is not defined - Events: 742 ~ Users: 544](https://sentry.io/organizations/gitcoin/issues/2445050840/?project=1398424&query=is%3Aunresolved+assigned_to%3Ame&statsPeriod=14d)

##### Testing

Tested locally

### CODE REVIEW

File: app/assets/v2/js/navbar.js [hunk 1]
```diff
@@ -186,18 +192,13 @@ const makeMenu = (navbarEl) => {
   // remove isVisible transitions to reduce jank
   const resetVisibility = () => {
     // removing isVisible will reset transition to just opacity
-    if (backgroundEl)
-      backgroundEl.classList.remove('isVisible');
-    if (contentEl)
-      contentEl.classList.remove('isVisible');
-    if (caretEl)
-      caretEl.classList.remove('isVisible');
+    backgroundEl.classList.remove('isVisible');
```
Comment (zacheryschiller) [inline app/assets/v2/js/navbar.js:195]: @thelostone-mc added these `if`s a few days ago to fix some of the Sentry bugs. Do we need to remove them? Or does this branch just need to be updated with those changes?

Comment (gdixon) [reply app/assets/v2/js/navbar.js:195]: This negates the need to make those checks -- they never fully resolved the issue, that function shouldn't be getting called so we don't need to check for the els there.

Comment (thelostone-mc) [reply app/assets/v2/js/navbar.js:195]: Makes sense ! if we can prevent it from being called at the start, that means my changes would be redundant

**Review by thelostone-mc — APPROVED** [review]

File: app/assets/v2/js/navbar.js (hunk 1 above)
Comment (zacheryschiller) [reply app/assets/v2/js/navbar.js:195]: Sounds good. Thanks for the extra info!

**Pull request merged**